env/env.common.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const rootDir = path.resolve(__dirname, '..');
+
+function rootPath(...args) {
+    return path.join(rootDir, ...args);
+}
+
+module.exports = {
+    path: rootPath,
+    staticServer: {
+        host: 'localhost',
```

Comment:
This is constants file. 

Could you please clarify, what you mean about constant?